### PR TITLE
rpc test chores

### PR DESCRIPTION
**Pull request by ouwenkg**
### Description of the Change

* Update `cita-vm` version to latest in Cargo.lock
* Handle returned evm error `StackUnderflow`
* Set `block_quota_limit` in `apply_transaction()`
* Some test script refactor
* Add necessary log record in critical point

### CODE REVIEW

File: cita-executor/core/src/libexecutor/block.rs [hunk 1]
```diff
@@ -137,9 +138,12 @@ impl ExecutedBlock {
         }
     }
 
-    #[allow(unknown_lints, clippy::too_many_arguments)] // TODO clippy
-    pub fn apply_transaction(&mut self, t: &SignedTransaction, conf: &BlockSysConfig) {
+    pub fn apply_transaction(&mut self, t: &SignedTransaction, sys_config: &GlobalSysConfig) {
```
Comment (kaikai1024) [inline cita-executor/core/src/libexecutor/block.rs]: Why change the `BlockSysConfig` to `GlobalSysConfig`

File: cita-executor/core/src/libexecutor/command.rs [hunk 2]
```diff
@@ -283,6 +282,7 @@ impl Commander for Executor {
             block_quota_limit: *header.quota_limit(),
             account_quota_limit: u64::max_value().into(),
         };
+        context.block_quota_limit = U256::from(self.sys_config.block_quota_limit);
```
Comment (kaikai1024) [inline cita-executor/core/src/libexecutor/command.rs]: It's so weird. Why not just set the block_quota_limit at the first.

File: cita-executor/core/src/libexecutor/executor.rs [hunk 3]
```diff
@@ -351,11 +350,11 @@ impl Executor {
     pub fn executed_result_by_height(&self, height: u64) -> ExecutedResult {
         let block_tag = BlockTag::Height(height);
         let sys_config = GlobalSysConfig::load(&self, block_tag);
-        let consensus_config = make_consensus_config(sys_config);
+        let consensus_config = make_consensus_config(sys_config.clone());
```
Comment (kaikai1024) [inline cita-executor/core/src/libexecutor/executor.rs]: Why add a clone?

File: cita-executor/core/src/libexecutor/command.rs (hunk 2 above)
Comment (ouwenkg) [reply cita-executor/core/src/libexecutor/command.rs]: The `block_quota_limit` in context from header is a default value. It should be updated. at first?

File: cita-executor/core/src/libexecutor/executor.rs (hunk 3 above)
Comment (ouwenkg) [reply cita-executor/core/src/libexecutor/executor.rs]: useless, I remove it.

File: cita-executor/core/src/libexecutor/block.rs (hunk 1 above)
Comment (ouwenkg) [reply cita-executor/core/src/libexecutor/block.rs]: I need to update `block_quota_limit` next.

**Pull request merged**